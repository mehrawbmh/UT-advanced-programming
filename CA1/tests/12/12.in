5 12:02
Banafsh 12
7
5 Modafe'an_e_Salamat 10 Tarbiat_Modarres 4 Boostan_e_Goftogoo 3 Borj_e_Milad 8 Meydan_e_Sa'adats 5 Shahid_Dadman 10 Meydan_e_Ketab
5
4 Shahid_Navab_e_Safavi 9 Roudaki 10 Komeil 4 Beryanak 3 Helal_Ahmar
Zard 6
4
4 Shademan 5 Doctor_Habibollah 12 Osatd_Moein 10 Meidan_e_Azadi
4
5 Meidan_e_enghelab_e_Eslami 4 teatr_e_Shahr 6 Ferdowsi 8 Darvaze_Dowlat
Abi 10
5
2 Fadak 10 Sabalan 12 Shahid_Madani 8 Emam_Hossein 14 Darvazeh_Shemran
4
3 Janbazan 4 Sarsabz 9 Daneshgah_e_Elm_o_San'at 10 Shahid_Bagheri
Soorati 20
10
2 Sarbaz 3 Shohadaye_Haftom_e_Tir 9 Shahid_Nejatollahi 12 Meydan_e_Hazrat_e_ValiAsr 6 Boostan_e_Laleh 4 Karegar 9 Shahrak_e_Azmayesh 7 Marzdaran 7 Yadegar_e_Emam 8 Shahid_Sattari
3
3 Meydan_e_Shohada 1 Ebn_e_Sina 4 Pirouzi
Ghermez 3
2
2 Khyyam 3 Panzdah_e_Khordad
1
10 Shoosh
Ebn_e_Sina
