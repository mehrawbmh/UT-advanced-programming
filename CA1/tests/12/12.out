Towards end of Soorati in 2 station(s)
12:24
1302
